cmd/test-configs/bob.toml

```diff
@@ -2,19 +2,18 @@
 
 usedurablestore = true
 
+guiport = 5007
```

Comment:
I think this formatting was done by this [toml extension](https://marketplace.visualstudio.com/items?itemName=be5invis.toml) in vs code. Seems pretty harmless so I'm going to include it.